src/components/introductory-block/introductory-block.tsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import classNames from "classnames/bind";
+
+import * as styles from "./introductory-block.module.scss";
+
+const cx = classNames.bind(styles);
+
+interface InputProps {
+  Icon: React.FC<React.SVGProps<SVGSVGElement>>;
+  gifAnimation: any;
```

Comment:
this is a bad habit. It is a good idea to setup eslint rule to avoid any types - https://typescript-eslint.io/rules/no-explicit-any/